src/sidebar/components/sidebar-content-omega.js

```diff
@@ -0,0 +1,168 @@
+import LoginPromptPanel from './login-prompt-panel';
+import SearchStatusBar from './search-status-bar';
+import SelectionTabs from './selection-tabs';
+import SidebarContentError from './sidebar-content-error';
+import ThreadList from './thread-list';
+
+/**
+ * Render the sidebar and its components
+ */
+function SidebarContent({
+  frameSync,
+  onLogin,
+  onSignUp,
+  loadAnnotationsService,
+  rootThread: rootThreadService,
+  streamer,
+}) {
+  const rootThread = useStore(store =>
+    rootThreadService.thread(store.getState())
+  );
+
+  // store state values
+  const focusedGroupId = useStore(store => store.focusedGroupId());
+  const hasAppliedFilter = useStore(store => store.hasAppliedFilter());
+  const isFocusedMode = useStore(store => store.focusModeEnabled());
+  const isLoading = useStore(store => {
+    return !store.hasFetchedAnnotations() || store.isFetchingAnnotations();
+  });
+  const isLoggedIn = useStore(store => store.isLoggedIn());
+  const linkedAnnotationId = useStore(store =>
+    store.directLinkedAnnotationId()
+  );
+  const linkedAnnotation = useStore(store =>
+    store.findAnnotationByID(linkedAnnotationId)
+  );
+  const searchUris = useStore(store => store.searchUris());
+  const sidebarHasOpened = useStore(store => store.hasSidebarOpened());
+  const userId = useStore(store => store.profile().userid);
+
+  // The local `$tag` of a direct-linked annotation; populated once it
+  // has anchored: meaning that it's ready to be focused and scrolled to
+  const linkedAnnotationAnchorTag =
+    linkedAnnotation && linkedAnnotation.$orphan === false
+      ? linkedAnnotation.$tag
+      : null;
+
+  // Actions
+  const clearSelectedAnnotations = useStore(
+    store => store.clearSelectedAnnotations
+  );
+  const selectTab = useStore(store => store.selectTab);
+
+  const hasDirectLinkedAnnotationError = useStore(store => {
+    if (isLoading || !linkedAnnotationId) {
+      return false;
+    }
+    return !store.annotationExists(linkedAnnotationId);
+  });
+
+  const hasDirectLinkedGroupError = useStore(store =>
+    store.directLinkedGroupFetchFailed()
+  );
+
+  const hasContentError =
+    hasDirectLinkedAnnotationError || hasDirectLinkedGroupError;
+
+  const showTabs = !hasContentError && !hasAppliedFilter;
+  const showSearchStatus = !hasContentError && !isLoading;
+
+  // Show a CTA to log in if successfully viewing a direct-linked annotation
+  // and not logged in
+  const showLoggedOutMessage = (() => {
+    if (
+      linkedAnnotationId &&
+      !isLoggedIn &&
+      !hasDirectLinkedAnnotationError &&
+      !isLoading
+    ) {
+      return true;
+    }
+    return false;
+  })();
+
+  const prevGroupId = useRef(focusedGroupId);
+
+  // Reload annotations when group, user or document search URIs change
+  useEffect(() => {
+    if (!prevGroupId.current || prevGroupId.current !== focusedGroupId) {
+      // Clear any selected annotations when the group ID changes
+      clearSelectedAnnotations();
```

Comment:
This is directly-ported logic, but I do wonder if the entire set of filters should be cleared, not just the selected annotations. Not urgent.

Reply:
That might depend on what the other filters are and whether it is appropriate to clear them when switching groups/user etc.